test/integration/patients/worklist/schedule.js

```diff
@@ -949,6 +967,11 @@ context('schedule page', function() {
       .find('.fa-square-check')
       .should('have.length', 20);
 
+    cy
+      .get('.schedule-list__table')
+      .find('.schedule-list__list-row .is-selected')
+      .should('have.length', 20);
+
```

Comment:
Verifies that the `.is-selected` class is added to each list item when the user selects-all via the bulk select controls.